routes/road-properties.js

```diff
@@ -270,6 +273,34 @@ function deleteHandler(req, res) {
   });
 }
 
+
+function patchPropertyHandler(req, res) {
+  if (validate(req.payload) !== undefined) {
+    return res(Boom.badData());
+  }
+
+  return knex('road_properties')
+    .select('id', 'properties')
+    .where({ id: req.params.road_id })
+  .then(function([response]) {
+    if (response === undefined) {
+      return res(Boom.notFound());
+    }
+
+    return knex('road_properties')
+      .where({ id: req.params.road_id })
+      .update({ properties: applyPatch(response.properties, req.payload).newDocument });
```

Comment:
Can't we do this without a library, using `Object.assign(response.properties, req.payload)` instead?

Or we could even rewrite both queries into just one, using the [JSONB concatenation operator](https://www.postgresql.org/docs/9.6/static/functions-json.html) (`||`), like:

```
knex.raw('
  UPDATE road_properties
  WHERE road_id = ${req.params.road_id}
  SET properties = properties || "${JSON.stringify(req.payload)}"::JSONB
')
```

Reply:
the PATCH spec isn't actually a matching JSON object of properties, but rather an array of operations: http://jsonpatch.com/

Reply:
Gotcha, sounds good, then! Svelte to have one handler take care of all the CRUD.